crux/src/Crux/Config/Common.hs

```diff
@@ -233,6 +237,8 @@ cruxOptions = Config
             section "proof-goals-fail-fast" yesOrNoSpec False
             "If true, stop attempting to prove goals as soon as one of them is disproved"
 
+          onlineProblemFeatures <- pure noFeatures
```

Comment:
Is the intention here to add a way to configure this, or just to allow downstream tools to twiddle this setting?

Reply:
The latter -- I'm not sure there would be a meaningful way to expose this feature except in clients of the library.

Reply:
I could imagine individual options for things like "enable nonlinear arithmetic during path-sat checking"... I'm not sure.  I guess this is fine, although it feels a little weird not to have it hooked up to any user settable options.